### DISCUSSION TITLE
Assign Roles

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#5385] [Q&A]
When i create new roles and groups, i assign it to specific user but on users page the column (Roles) not show the role name ONLY the super admin appears and after i login with this user it gives me unauthorized and logout. How to fix this to show the role name and fix unauthorized issue

**Comment by HazemMohamed252** [comment]
Any help ?

**Reply by edalzell** [reply]
I don't understand what you mean, can you include some screenshots that show what you're talking about?
